packages/mask/src/index.js

```diff
@@ -1,9 +1,10 @@
-
 export default function (Alpine) {
     Alpine.directive('mask', (el, { value, expression }, { effect, evaluateLater }) => {
         let templateFn = () => expression
         let lastInputValue = ''
 
+        let nextTick = callback => requestAnimationFrame(() => requestAnimationFrame(callback))
```

Comment:
Alpine exposes `nextTick` on the `Alpine` object.

Reply:
Done